aries_cloudagent/protocols/coordinate_mediation/v1_0/handlers/keylist_request_handler.py

```diff
@@ -0,0 +1,49 @@
+"""Handler for incoming route-list-request messages."""
+
+from .....storage.error import StorageNotFoundError
+from ....problem_report.v1_0.message import ProblemReport
+
+from .....messaging.base_handler import (
+    BaseHandler,
+    BaseResponder,
+    HandlerException,
+    RequestContext,
+)
+
+from ..manager import MediationManager as Manager
+from ..messages.keylist_query import KeylistQuery as request
```

Comment:
Types should be camel case ("Request" instead of "request"); this looks like a variable, otherwise

Reply:
good catch.